tests/models/gemma/test_modeling_gemma.py

```diff
@@ -788,3 +789,24 @@ def test_model_7b_4bit(self):
         output_text = tokenizer.batch_decode(output, skip_special_tokens=True)
 
         self.assertEqual(output_text, EXPECTED_TEXTS)
+
+    def test_model_2b_bf16_dola(self):
+        model_id = "google/gemma-2b"
+        EXPECTED_TEXTS = [
+            "Hello I am doing an experiment and need to get the mass of a block. The problem is, it has no scale",
+            "Hi today we have the review for a <strong>2016/2017</strong> season of",
+        ]
```

Comment:
Are these (and the other model tests) the same results you obtain in your original implementation? If so, let's add a comment on top of `EXPECTED_TEXTS`

Reply:
Added!